pkg/asset/asset.go

```diff
@@ -18,8 +19,14 @@ type Asset interface {
 
 	// Name returns the human-friendly name of the asset.
 	Name() string
+
+	// Load returns the on-disk the asset if they exist.
+	Load(FilePatternFetcher) (found bool, err error)
```

Comment:
If you move `Load` to a separate interface (or to `WriteableAsset`), then you wouldn't need to add no-op functions to assets that do not write files to disk.

Reply:
SG